.editorconfig

```diff
@@ -14,6 +14,87 @@ indent_size = 4
 [project.json]
 indent_size = 2
 
+# C# files
+[*.cs]
```

Comment:
For `csharp_preserve_single_line_blocks = true`, it will result in:
```C#
public int Foo { get; set; }
```

For `csharp_preserve_single_line_blocks = false`, it will result in:
```C#
public int Foo
{
    get;
    set;
}
```

For `csharp_preserve_single_line_statements = true`, it will result in:
```C#
int i = 0; string name = "John";
```

For `csharp_preserve_single_line_statements = false`, it will result in:
```C#
int i = 0;
string name = "John";
```